products/samsungmobile.md

```diff
@@ -812,29 +815,29 @@ releases:
 -   releaseCycle: "galaxy-m21s"
     releaseLabel: "Galaxy M21s"
     releaseDate: 2020-11-06
-    eoas: false
-    eol: false
+    eoas: true
+    eol: true
```

Comment:
```suggestion
    eoas: 2022-11-06
    eol: 2024-11-06
```

https://androidspotlight.com/software-update-policy-for-every-samsung-device/, guessed EOL